projects/packages/forms/src/icons/akismet.tsx

```diff
@@ -46,4 +46,4 @@ const AkismetSvg = ( props: SVGProps & { width?: number; height?: number } ) =>
 	);
 };
 
-export default AkismetSvg;
```

Comment:
One small difference I see is on the Integrations panel, where the Akismet icon is now square vs circular. Not a big deal to me, and not sure if that's intended, but it is circular on trunk. 

<img width="287" alt="integrations-panel-akismet" src="https://github.com/user-attachments/assets/bca9c3c1-e9b7-4bb0-b8b7-da3b0726e1d7" />